web/src/main/java/org/cbioportal/web/config/CustomObjectMapper.java

```diff
@@ -28,6 +80,5 @@ public CustomObjectMapper() {
         mixinMap.put(Sample.class, SampleMixin.class);
         mixinMap.put(TypeOfCancer.class, TypeOfCancerMixin.class);
         super.setMixInAnnotations(mixinMap);
-        super.configure(JsonGenerator.Feature.WRITE_NUMBERS_AS_STRINGS, true);
```

Comment:
@ersinciftci @sheridancbio I think this might cause another issue besides the data type mismatch in unit test. Did you check is there anyone who is using the API and expect values to be string? After we remove WRITE_NUMBERS_AS_STRINGS setting, it will return number, which might make break some current implementation.